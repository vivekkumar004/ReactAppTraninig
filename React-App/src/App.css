nav {
  position: fixed;
  top: 0;
  right: 0;
  left: 0;
  padding: 1rem 3rem;
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: var(--foregroundColor);
}

nav img {
  width: 50px;
  height: 50px;
}

.heading {
  display: flex;
  align-items: center;
}

.heading h1 {
  margin-right: 1rem;
}