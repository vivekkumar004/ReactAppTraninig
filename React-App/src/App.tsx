import "./App.css";
import List from "./components/List";

function App({ firstName, age }: any) {
  return (
    <div>
      <h1>
        Welcome to {firstName} {age} training!
      </h1>
      <List />
    </div>
  );
}

export default App;