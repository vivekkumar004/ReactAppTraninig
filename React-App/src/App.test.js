import { title } from "./App";

describe("unit testing basics", () => {
  test("assert if title is React", () => {
    expect(true).toBe(true);
  });

  test("assert if title is React", () => {
    expect(title).toBe("React Training");
  });
});