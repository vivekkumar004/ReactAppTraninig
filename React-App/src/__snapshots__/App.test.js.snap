// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Input with label component snapshot testing 1`] = `
<div>
  <label>
    Search Label
     
  </label>
  <input
    type="text"
    value="React"
  />
</div>
`;
